data-release-pipeline/prepare_for_molgenis/main.py

```diff
@@ -0,0 +1,155 @@
+import os
+import re
+import sys
+from shutil import copyfile
+
+from ArgumentParser import ArgumentParser
+from ConfigWriter import ConfigWriter
+from GithubConnector import GithubConnector
+from GithubRepoVersionChecker import GithubRepoVersionChecker
+from ScriptUtils import ScriptUtils
+
+github = GithubConnector()
+version_checker = GithubRepoVersionChecker()
+repo = 'molgenis-py-consensus'
+
+done_msg = '✅ Done'
+
+
+def retrieve_molgenis_py_consensus(vkgl_dir):
+    print('Retrieving newest molgenis-py-consensus...', end=' ')
```

Comment:
`run.sh` works with a fixed version of `data-transform-vkgl` to ensure that results can be reproduced. this should work the same for `molgenis-py-consensus`. otherwise results might not be reproducible and in time older versions of run.sh will fail to run when major changes are made to `molgenis-py-consensus`.

Reply:
In most cases you want to use the most recent version, of all tools. `run.sh` is linked to `data-transform-vkgl` because the code is in the same repository. There it makes more sense to link the versions. `molgenis-py-consensus` however is a complete separate repository. I think removing this automatic step will lead to unnecessary manual work. It also means that whenever you update `molgenis-py-consensus` the `data-transform-vkgl` repository always needs to be updated as well. Most of the time when you alter one of those repositories, you don't need to alter the other. So this is also an additional step in the process that complicates things. I however think that it is necessary to have the option to select a version for reproduction purposes.